mbta.js

```diff
@@ -0,0 +1,77 @@
+
+
+const subwayLines = {
+    'Red': ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife'],
+    'Green': ['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore '],
+    'Orange': ['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crocssing', 'Chinatown', 'Back Bay', 'Forest Hills']
+};
+
+const stopsBetweenStations = function (startLine, startStation, endLine, endStation) {
+  
+     startLine = lettersCase(startLine);
+    startStation = lettersCase(startStation);
+    endLine = lettersCase(endLine);
+    endStation = lettersCase(endStation);
+    
+    if (startLine === endLine) {
+       let startIndex = subwayLines[startLine].indexOf(startStation)
+        let endIndex = subwayLines[endLine].indexOf(endStation)
+        let stop = Math.abs( startIndex - endIndex ) 
+        return stop;
+
+    } 
+    
+    else {
+
+        let ofPark = subwayLines[startLine].indexOf('Park Street')
+        let firstStop = subwayLines[startLine].indexOf(startStation)
+
+        let firstPark = Math.abs( firstStop - ofPark )
+        let indexFinalStop = subwayLines[endLine].indexOf(endStation)
+        let finalPark = Math.abs(ofPark - indexFinalStop )
```

Comment:
"ofPark" you should have 2 of them, one for the "Park State" index in the beginning at the start line, and another for the final's stop's line.